### PR TITLE
- add revert target rate to LO screen

### PR DESCRIPTION
**Pull request by MrQKyber**
(cherry picked from commit 052ae626dca990240716a48ac16d245aad372a51)

### CODE REVIEW

File: KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewModel.swift [hunk 1]
```diff
@@ -299,13 +299,29 @@ class KNCreateLimitOrderViewModel {
     return "1 \(self.toSymbol) = \(rateText) \(self.fromSymbol)"
   }
 
+  var resertTargetExchangeRateString: String? {
```
Comment (manhlx3006) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewModel.swift]: typo: resert
here we should check if the source token is a quote token, if it is not, no need to display revert rate

Comment (MrQKyber) [reply KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewModel.swift]: validation for check if token is quote or not is placed in displayTargetExchangeRate

File: KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift [hunk 2]
```diff
@@ -617,6 +618,7 @@ extension KNCreateLimitOrderViewController {
   fileprivate func updateCurrentMarketRateUI() {
     self.currentRateLabel.text = "\(self.viewModel.displayCurrentExchangeRate)"
     self.compareMarketRateLabel.attributedText = self.viewModel.displayRateCompareAttributedString
+    self.targetReverseRateLabel.text = viewModel.displayTargetExchangeRate
```
Comment (manhlx3006) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift]: It is not correct to put here, the target reverted rate is calculated using user's target rate.
So it should be updated whenever the user's target rate changed

File: KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift [hunk 3]
```diff
@@ -613,7 +614,16 @@ extension KNCreateLimitOrderViewController {
     self.view.layoutIfNeeded()
   }
 
+  fileprivate func updateTargetRateUI(_ text: String) {
+    viewModel.updateTargetRate(text)
+    updateRevertTargetRateUI()
+  }
+
   // Update current martket rate with rate from node or cached
+  fileprivate func updateRevertTargetRateUI() {
```
Comment (manhlx3006) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift]: No need to have another function, just put it inside `updateTargetRateUI`

File: KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift [hunk 4]
```diff
@@ -519,7 +520,7 @@ class KNCreateLimitOrderViewController: KNBaseViewController {
     guard let rate = self.viewModel.rateFromNode else { return }
     let rateDisplay = rate.displayRate(decimals: self.viewModel.to.decimals).removeGroupSeparator()
     self.targetRateTextField.text = rateDisplay
-    self.viewModel.updateTargetRate(rateDisplay)
+    updateTargetRateUI(rateDisplay)
```
Comment (manhlx3006) [inline KyberNetwork/KyberNetwork/Screens/Tabbars/LimitOrder/Controllers/CreateOrder/KNCreateLimitOrderViewController.swift]: Please use `self` here for consistency

**Pull request merged**